src/styles/main.scss

```diff
@@ -1 +1,71 @@
-// put code here
+@import "variable";
+
+@keyframes rotate-minute {
+  0% {
+    transform: translateX(-50%) rotate(0deg);
+  }
+  100% {
+    transform: translateX(-50%) rotate(360deg);
+  }
+}
+
+@keyframes rotate-hour {
+  0% {
+    transform: translateX(-50%) rotate(-90deg);
+  }
+  100% {
+    transform: translateX(-50%) rotate(270deg);
+  }
+}
```

Comment:
It is necessary to use only one animation, since the arrows rotate according to the same script
```suggestion
@keyframes rotate {
  0% {
    transform: translateX(-50%) rotate(0deg);
  }
  100% {
    transform: translateX(-50%) rotate(360deg);
  }
}
```